### ISSUE TITLE
test code in test-config.json

### ISSUE THREAD

**Issue by yanj-github** [cta-wave/dpctf-tests#25]
Will you be able to change the following from the test-config.json please?
From
"regular-playback-of-chunked-content-non-aligned-manual.html"
to 
**"regular-playback-of-chunked-content-non-aligned-append-manual.html"**

regular-playback-of-chunked-content-non-aligned-append-manual is used in tests.json but this is different on test-config.json.
It would be better to keep the test codes consistent.

**Comment by FritzHeiden** [comment]
Yes, I will change the names so they are consistent with the actual file names
